test/unit/idex/identity-resolver.spec.ts

```diff
@@ -37,15 +38,15 @@ describe('IdentityResolver', () => {
 
   it('should invoke callback on success, store the result in a cookie', function (done) {
     const response = { id: 112233 }
-    const identityResolver = IdentityResolver({}, storageHandler, calls, eventBus)
+    const identityResolver = IdentityResolver.make({}, storageHandler, calls, eventBus)
     const successCallback = (responseAsJson) => {
       expect(callCount).to.be.eql(1)
       expect(errors).to.be.empty()
       expect(responseAsJson).to.be.eql(response)
       expect(requestToComplete.url).to.eq('https://idx.liadm.com/idex/unknown/any')
       expect(responseAsJson).to.be.eql(response)
       expect(callCount).to.be.eql(1)
-      expect(storageHandler.getCookie('__li_idex_cache')).to.be.eq(JSON.stringify(response))
+      expect(storageHandler.getCookie('__li_idex_cache_e30')).to.be.eq(JSON.stringify(response))
```

Comment:
The cache key for no provided additional parameters changed. (As we are using `{}` instead of `undefined`)

I don't think this is a problem as we will just re-resolve, but I can revert this change if desired

Reply:
I think it is fine.